### PR TITLE
Cleaning up client lists and API titles

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
Remember to click on the "View" button to make sure the rendered README.md files look right :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,39 +14,41 @@ Java idiomatic client for [Google Cloud Platform][cloud-platform] services.
...
 -  [BigQuery Data Transfer](google-cloud-clients/google-cloud-bigquerydatatransfer) (Beta)
 -  [Cloud Data Loss Prevention](google-cloud-clients/google-cloud-dlp) (Beta)
--  [Stackdriver Error Reporting](google-cloud-clients/google-cloud-errorreporting) (Beta)
 -  [Cloud Firestore](google-cloud-clients/google-cloud-firestore) (Beta)
+-  [Cloud IoT Core](google-cloud-clients/google-cloud-iot) (Beta)
 -  [Cloud KMS](google-cloud-clients/google-cloud-kms) (Beta)
 -  [Cloud Spanner](google-cloud-clients/google-cloud-spanner) (Beta)
+-  [Cloud Speech](google-cloud-clients/google-cloud-speech) (Alpha)
 -  [Cloud Video Intelligence](google-cloud-clients/google-cloud-video-intelligence) (Beta)
+-  [Cloud Text-to-Speech](google-cloud-clients/google-cloud-texttospeech) (Beta)
```
Comment (hzyi-google) [inline README.md]: Nit: `Text-to-Speech` above `Video Intelligence`?

File: google-cloud-clients/google-cloud-websecurityscanner/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Google Cloud Java Client for Web Security Scanner
```
Comment (hzyi-google) [inline google-cloud-clients/google-cloud-websecurityscanner/README.md]: Nit: Cloud Security Scanner?

**Comment by hzyi-google** [comment]
LGTM except for the two trivial issues

File: README.md (hunk 1 above)
Comment (garrettjonesgoogle) [reply README.md]: Done

File: google-cloud-clients/google-cloud-websecurityscanner/README.md (hunk 2 above)
Comment (garrettjonesgoogle) [reply google-cloud-clients/google-cloud-websecurityscanner/README.md]: Done

**Pull request merged**